src/lib/airtable/interface.ts

```diff
@@ -23,13 +23,13 @@ export type Row = {
 export interface UserRecord {
   id: string;
   username: string;
-  email: string;
-  photo?: Airtable.Attachment[];
-  siteIds: string[];
-  sites: SiteRecord[];
-  password: string;
-  name: string;
   admin: boolean;
+  email?: string;
+  photo?: Airtable.Attachment[];
+  siteIds?: string[];
+  sites?: SiteRecord[];
+  password?: string;
```

Comment:
Can you put a TODO that we should scrub password out? There's no reason to expose it. We'd eventually want to just not deliver to frontend via a `restrictColumn` thing on the write access resolver but right now `login` doesn't use the write access resolver